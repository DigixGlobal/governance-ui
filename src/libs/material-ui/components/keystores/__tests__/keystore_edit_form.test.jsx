import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import KeystoreEditForm from '../keystore_edit_form';

jest.mock('../../../keystoreTypes', () => ({
  getKeystoreComponent: () => jest.fn,
}));

const data = {
  type: {
    id: 'testId',
  },
};

describe('<KeystoreEditForm />', () => {
  test.skip('renders correctly', () => {
    const component = shallow(
      <KeystoreEditForm data={data} />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
