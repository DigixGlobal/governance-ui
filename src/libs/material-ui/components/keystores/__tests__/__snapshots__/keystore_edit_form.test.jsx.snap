// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<KeystoreEditForm /> renders correctly 1`] = `
<FormField>
  <KeystoreTypeMessage
    keystoreType={
      Object {
        "id": "testId",
      }
    }
  />
  <[object Object]
    data={
      Object {
        "type": Object {
          "id": "testId",
        },
      }
    }
  />
</FormField>
`;
